libqtile/backend/wayland/core.py

```diff
@@ -108,6 +108,39 @@
     from libqtile.core.manager import Qtile
 
 
+class ImplicitGrab:
```

Comment:
Might be worth using `HasListeners` just to follow the style of other classes that uses listeners?

Reply:
Sure, if you prefer.